### ISSUE TITLE
bug(ui): stage phase icon + tooltip missing for some phases

### ISSUE THREAD

**Issue by krancour** [akuity/kargo#2776]
In this particular case, the Stage was in the new (as of #2762) `Failed` phase, so the UI not being prepared for that wasn't surprising, but I believe I've seen this occur many times prior to #2762. (I have a hunch that _no phase_ (`""`), which is possible for a brand new Stage, may be unaccounted for.) 

![Screenshot 2024-10-17 at 4 04 18 PM](https://github.com/user-attachments/assets/7b26f753-d08b-4bd9-878a-dd3677bf9cfe)

**Comment by Marvin9** [comment]
@krancour thats Stage health status icon. Ok this UI code was a year ago and I think there was not "phase" initially? Or maybe health and phase had some correlation?

Anyways this needs to be changed if thats the case.

**Comment by krancour** [comment]
@Marvin9 you're right. I have my wires crossed here. That icon _is_ for health, and _not_ for phase, as you correctly point out.

Health can legitimately be nil, especially for a brand new Stage that has not had any Promotions yet (or hasn't had any _successful_ Promotions yet).

I wonder if nil health is the source of this.

**Comment by Marvin9** [comment]
Yes nil health is the issue. But I think we should show both.. stage health + stage phase right?

**Comment by Marvin9** [comment]
I believe UI can use new analogy after https://github.com/akuity/kargo/pull/2905?
